cosmos/precompile/bank/helpers.go

```diff
@@ -42,8 +42,8 @@ func sdkCoinsToEvmCoins(sdkCoins sdk.Coins) []generated.IBankModuleCoin {
 	return evmCoins
 }
 
-// extractCoinsFromInput converts coins from input (of type any) into sdk.Coins.
-func extractCoinsFromInput(coins any) (sdk.Coins, error) {
+// ExtractCoinsFromInput converts coins from input (of type any) into sdk.Coins.
```

Comment:
I would move these coins conversion helper functions to the cosmos/lib folder in a file called coins.go or similar